server/data/quotes.json

```diff
@@ -570,5 +570,9 @@
   {
     "author": "William Shakespeare",
     "sentence": "My words fly up, my thoughts remain below. Words without thoughts never to heaven go."
-  }
+  },
+  {
+    "author": "Zadie Smith",
+    "sentence": "The past is always tense, the future perfect."
+  },
```

Comment:
Also apart from @Kvaibhav01’s comment, please remove the extra comma here as it is an invalid JSON

Reply:
I think I managed to correct the pr